groups/chips-db/variables.tf

```diff
@@ -264,3 +264,13 @@ variable "netapp_ips" {
   type        = list(string)
   description = "List of Netapp IP addresses to use for iscsi discovery."
 }
+
+
+# ------------------------------------------------------------------------------
+# CHIPS DB SG variables
+# ------------------------------------------------------------------------------
+variable "chips_db_sg" {
+  type        = list(any)
+  description = "List of CHIPS DB SGs"
```

Comment:
SGs should be security groups in the description, make it easier for others to understand.

Reply:
Changed to Security Groups